### PR TITLE
Clean up init_run() docstring

### PR DESCRIPTION
**Pull request by normandy7**
- Remove reST markup
- Omit type and defaults from arg list
- Change init() to init_run()
- Adjust code examples (include resuming a run)
- Spelling and grammar

### CODE REVIEW

**Comment by normandy7** [comment]
> This pull request removes 700 and adds 477 tests. Note that renamed tests count towards both.

What happened here? 😅

File: neptune/new/internal/init/run.py [hunk 1]
```diff
@@ -99,102 +99,107 @@ def init_run(
...
-        fail_on_exception (bool, optional): Whether to register an uncaught exception handler to this process and,
-            in case of an exception, set run's sys/failed to True. Exception is always logged
-        monitoring_namespace (str, optional): Namespace inside which all monitoring logs be stored.
+        project: Name of the project where the run should go, in the form "workspace-name/project_name".
+        If None (default), the value of the NEPTUNE_PROJECT environment variable is used.
+        api_token: User's API token. Defaults to None.
+            If None (default), the value of the NEPTUNE_API_TOKEN environment variable is used.
+            Note: To keep your API token secure, save it to the NEPTUNE_API_TOKEN environment variable rather than
+            placing it in plain text in the source code.
+        with_id: If you want to resume a run, the identifier of the existing run.
```
Comment (Herudaio) [inline neptune/new/internal/init/run.py:113]: Where is run param? It's still there so even if deprecated maybe we should have it with a proper info?

File: neptune/new/internal/init/run.py [hunk 2]
```diff
@@ -99,102 +99,107 @@ def init_run(
...
+        description: Editable description of the run. Defaults to `''`.
+            The description is displayed in the run details and can be added to the runs table as a column.
+        tags: Tags of the run as a list of strings. Defaults to `[]`.
+            Tags are displayed in the run details and in the runs table as a column.
+            You can edit the tags after the run is created, either through the app or the API.
+        source_files: List of source files to be uploaded.
+            Uploaded source files are displayed in the 'Source code' tab of the run view.
+            Unix style pathname pattern expansion is supported. For example, you can pass `*.py` to upload
+            all Python files from the current directory.
+            If None is passed, the Python file from which the run was created will be uploaded.
```
Comment (Herudaio) [inline neptune/new/internal/init/run.py:133]: People often ask about turning this off and solution is empty list [], maybe we can add it to the description?

File: neptune/new/internal/init/run.py [hunk 3]
```diff
@@ -99,102 +99,107 @@ def init_run(
...
-        flush_period (float, optional): In an asynchronous (default) connection mode how often asynchronous thread
-            should synchronize data with Neptune servers. Defaults to 5.
-        proxies (dict of str, optional): Argument passed to HTTP calls made via the Requests library.
-            For more information see
-            `their proxies section <https://2.python-requests.org/en/master/user/advanced/#proxies>`_.
-        capture_traceback (bool, optional):  Whether to send run’s traceback in case of an exception.
-            Defaults to `True`.
-            Tracked metadata will be stored inside `monitoring/traceback`.
+        flush_period: In the asynchronous (default) connection mode, in seconds, how often the asynchronous thread
+            should synchronize data with Neptune servers. Defaults to 5 (every 5 seconds).
```
Comment (Herudaio) [inline neptune/new/internal/init/run.py:147]: Potential improvement - there were recently some questions from users that understood here that they can use this to lower frequency of sending info to Neptune and save on monitoring hours and I think the description did led them to this conclusion. More info -> Prince -> Adam Nieżurawski

**Review by Herudaio — COMMENTED** [review]
@normandy7 no other comments. Apart from `run` param I think the rest can be discarded if you don't think it's a priority right now.

File: neptune/new/internal/init/run.py (hunk 2 above)
Comment (normandy7) [reply neptune/new/internal/init/run.py:133]: Good idea

File: neptune/new/internal/init/run.py (hunk 1 above)
Comment (normandy7) [reply neptune/new/internal/init/run.py:113]: You're right

**Review by Raalsky — APPROVED** [review]

**Pull request merged**